Equinox/ModuleSettings.cpp

```diff
@@ -0,0 +1,30 @@
+﻿#include "ModuleSettings.h"
+#include "parson.h"
+
+ModuleSettings::ModuleSettings(bool start_enabled) : Module(start_enabled)
+{
+}
+
+ModuleSettings::~ModuleSettings()
+{
+}
+
+bool ModuleSettings::Init()
+{
+	settings = json_value_get_object(json_parse_file(inputFile));
+
+	if(settings != nullptr)
+	{
+		MaxFps = static_cast<int>(json_object_get_number(settings, "maxFps"));
+		if (MaxFps == 0)
```

Comment:
Instead of checking for 0 (it is a possible config value), we should check if the value exists with **json_object_has_value** or **json_object_has_value_of_type**